core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -518,6 +534,15 @@ func saveSentAttempt(db *gorm.DB, attempt *models.EthTxAttempt) error {
 	return errors.Wrap(db.Save(attempt).Error, "saveSentAttempt failed")
 }
 
+func saveInsufficientEthAttempt(db *gorm.DB, attempt *models.EthTxAttempt) error {
```

Comment:
Would this function be better suited in the orm?

Reply:
Generally speaking I prefer SQL to be close to the code that requires it. This makes it easier to read, grep etc because all the logic is together.

ORM is good for general queries that are used all over the codebase.